### DISCUSSION TITLE
CAN-BUS Shield V2.0 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#212] [Q&A]
# CAN-BUS Shield V2.0 | Seeed Studio Wiki

CAN-BUS_Shield_V2.0

https://wiki.seeedstudio.com/CAN-BUS_Shield_V2.0/#schematic-online-viewer

**Comment by tthom289** [comment]
On the V2 CAN-BUS shield what type of connectors are used for the I2C & A0?

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by romeroej-inch** [comment]
Im trying to use the shield with a NRF52840 DK board.  Has anybody had any luck?

Is this board able to work on 3.3v?

**Comment by ke5eua** [comment]
Is there an updated pin map for the V2 can bus?

**Reply by nfs0619** [reply]
Hi there, you can find the pin map for the CAN-BUS Shield V2.0 here:
[https://wiki.seeedstudio.com/CAN-BUS\_Shield\_V2.0/#hardware-overview](https://wiki.seeedstudio.com/CAN-BUS_Shield_V2.0/#hardware-overview)

If you have any questions or need further clarification, feel free to contact our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)**. They can provide faster and more detailed assistance.
